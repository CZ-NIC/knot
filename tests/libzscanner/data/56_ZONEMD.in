$ORIGIN	.
$TTL	1

; OK
@	ZONEMD	0	0	0	00		; The simplest variant
@	ZONEMD	4294967295 255	255	00		; Maximal numbers
@	ZONEMD	0	0	0	0102 00 FF	; Hex string with blank spaces inside
@	TYPE53	\# 7 00000000000000			; TYPE + Hexadecimal rdata
@	TYPE63	0       0       0       00		; TYPE
@	zonemd	0       0       0       00		; Type in lower-case

; KO
@	ZONEMD
@	ZONEMD						; Empty rdata
@	ZONEMD	\# 0					; Hex empty rdata
@	ZONEMD	4294967296 0	0	00		; Serial overflow
@	ZONEMD	0	256	0	00		; Type overflow
@	ZONEMD	0	0	256	00		; Reserved overflow
@	ZONEMD	0	0	0	0		; Hex block length must be multiple of 2
@	ZONEMD	0	0	0	0x		; Bad hex char
@	ZONEMD	0	0	0			; Missing item
