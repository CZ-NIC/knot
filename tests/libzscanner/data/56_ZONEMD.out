OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003F
RDATA=00000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003F
RDATA=FFFFFFFFFFFF00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003F
RDATA=000000000000010200FF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0035
RDATA=00000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003F
RDATA=00000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003F
RDATA=00000000000000
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER32_OVERFLOW
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_BAD_HEX_CHAR
------
